.github/workflows/release.yaml

```diff
@@ -0,0 +1,52 @@
+name: Weekly Release
+
+on:
+  schedule:
+    - cron: '0 11 * * 1' # Run every Monday at 7am NYC time
+      branches:
+        - master
+  workflow_dispatch: # allows manual releasing
+      branches:
+        - master
+jobs:
+  build_and_release:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+
+      - name: Set up Bazel
+        uses: actions/setup-java@v3
+        with:
+          distribution: 'adopt'
+          java-version: '21'
+
+      - name: Build with Bazel
+        run: bazel build java/com/google/copybara/copybara_deploy.jar
+
+      - name: Get current date
+        id: date
+        run: echo "::set-output name=date::$(date +%Y%m%d)"
```

Comment:
this is deprecated https://github.blog/changelog/2022-10-11-github-actions-deprecating-save-state-and-set-output-commands/
they want you to  use GITHUB_OUTPUT instead